### PR TITLE
Invitation Email for new Users

### PR DESCRIPTION
**Pull request by chrysalag**
## Story
https://trello.com/c/7kXpgmQO/28-as-a-user-i-want-an-invitation-mail-and-not-a-password-reset-mail

## Goals
- [x] Story is tested
- [x] Acceptance criterias are met
- [ ] Code is reviewed
- [x] Travis is green
- [x] Story is on resolved in trello
- [ ] Readme is updated if needed
- [ ] Styled with Bootstrap v3
- [ ] Mobile view is usable
- [ ] Seeds are available
- [ ] German translations available

### CODE REVIEW

File: config/initializers/devise.rb [hunk 1]
```diff
@@ -116,6 +116,54 @@
   # Send a notification email when the user's password is changed.
   # config.send_password_change_notification = false
 
+  # ==> Configuration for :invitable
+  # The period the generated invitation token is valid, after
+  # this period, the invited resource won't be able to accept the invitation.
+  # When invite_for is 0 (the default), the invitation won't expire.
+  config.invite_for = 1.day
```
Comment (kitsane) [inline config/initializers/devise.rb]: Just comment this out, the default should be ok

File: app/views/devise/invitations/edit.html.slim [hunk 2]
```diff
@@ -0,0 +1,14 @@
+h2
```
Comment (kitsane) [inline app/views/devise/invitations/edit.html.slim]: Look at the current form views and adapt the layout to them

File: app/views/devise/invitations/new.html.slim [hunk 3]
```diff
@@ -0,0 +1,12 @@
+h2
```
Comment (kitsane) [inline app/views/devise/invitations/new.html.slim]: same as above

**Comment by chrysalag** [comment]
Andi told me that I can merge it.

**Pull request merged**